integration-tests/tests/find-workflow.js

```diff
@@ -7,7 +7,7 @@ const { withUserToken } = require('../utils/terra-sa-utils')
 
 
 const testFindWorkflowFn = _.flow(
-  withWorkspace,
+  withWorkspace(),
```

Comment:
This is to illustrate how tests would need to be modified if they don't care about customizing `bucketLocation`.